sdk/keyvault/azure-security-keyvault-secrets/README.md

```diff
@@ -16,7 +16,7 @@ Maven dependency for the Azure Key Vault Secrets client library. Add it to your
 <dependency>
     <groupId>com.azure</groupId>
     <artifactId>azure-security-keyvault-secrets</artifactId>
-    <version>4.3.0-beta.4</version>
+    <version>4.3.0-beta.5</version>
```

Comment:
Is this version released?

Reply:
Please refer to [this comment](https://github.com/Azure/azure-sdk-for-java/pull/20521#discussion_r610940121).